web/pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx

```diff
@@ -753,6 +753,7 @@ export class ResultSetUtils {
       }
       this.eventBus.fireEvent(QUERY_TOOL_EVENTS.SET_MESSAGE, tabMsg, true);
       this.eventBus.fireEvent(QUERY_TOOL_EVENTS.FOCUS_PANEL, PANELS.MESSAGES);
+      this.eventBus.fireEvent(QUERY_TOOL_EVENTS.SET_QUERY_DATA, null);
```

Comment:
What is this for, no comments added? And why do we need an event in the same component, which is not used in any other component?

Reply:
In today's scrum call I discussed with the team about this issue Maybe you left the call.
It is a bug like:-
when we execute any `select` SQL it fetches the data and shows it in the data output tab
And then if we run `commit or alter`  SQL, it shows us a message, but still the data-output tab has the old `select` query value.
That's why I have made these changes. Currently, we are not clearing the data-output tab if a query does not have any result to display.

Reply:
It was discussed in a call, but how will someone know when reading the code? Add some comments.